### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Public Websites, VA.gov, Federal Standardized Header/Footer

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#71524]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Should:** I have some concerns about the "USA Government" color choices at the top of the page. When testing with a [colorblindness simulator](https://wearecolorblind.com/resources/colorblindly-colorblindness-simulator/), the blue-on-black and the red-on-black are potentially at risk of getting lost for some users.

   The [blue-on-black has a contrast ratio of 4.9:1](https://webaim.org/resources/contrastchecker/?fcolor=3975EA&bcolor=000000), and the [red-on-black has a contrast ratio of 5.45:1](https://webaim.org/resources/contrastchecker/?fcolor=FF2000&bcolor=000000). That meets WCAG AA minimums, but it doesn't clear that bar by a ton and it doesn't meet the AAA requirement of 7:1. (White-on-black is 21:1, for reference.)
   
   So this meets minimum contrast requirements. But for something as important as this --- establishing trust that this is a legitimate website speaking for the federal government --- I think it might be worth aiming higher than minimums. I don't have specific recommendations here, except to pay close attention to comments on color perception during user research.
- **Implementation note:** When coding this, make sure that the VA seal is legible up to 400% zoom. On the current VA.gov, the VA seal in the logo disappears above 200% zoom, but the seal in the footer persists at high zoom and is a little pixelated once you get past 200%. The header seal in the Sketch file seems fine at high zoom, but just double-check to make sure whatever gets included in the coded version is sufficiently high-resolution up to 400%.
- **Consider:** Echoing Erin in #71505, the crisis line button moves from the third tab stop to (by my count) the ninth tab stop. This change could impact keyboard-only users in crisis --- especially screen reader users who don't get the visual affordance of the red button standing out against the background. On the current VA.gov, the crisis line's location in the tab order ensures that it's one of the first things a blind user will encounter when exploring the header. With this change, it looks like it will be the last thing in the header they encounter before reaching the main content of the page.
   
   If that's a deliberate change to de-emphasize the crisis line, that's fine. But if it's not a deliberate change, I think it's worth exploring the potential consequences. In other words, I'm not saying "don't move this button!" but I am asking "are you sure you want to move this button?" This may also be something worth exploring in research, if possible.
- **Should:** In addition to mobile, now would be a great time to plan for what the header and footer look like in Windows High Contrast Mode (dark and light), when iPhone users turn on Invert Colors, etc.

**Comment by aklausmeier** [comment]
@laflannery Let's discuss when we meet tomorrow

**Comment by laflannery** [comment]
1 (USA logo) and 3 (VCL Button) have been noted and will be discussed again after research, depending on what comes out of that as it will inform the decisions made here.

2 - the size of the logo will be added as an annotation so that we implement a logo large enough to not be pixelated

**Comment by laflannery** [comment]
@briandeconinck Added my notes on your feedback in the comment above, I'm going to close this but let me know if you have any further questions or this needs to be reopened for any reason. Thanks!

CCL @aklausmeier @FranECross
